examples/ASDKgram/Sample/PhotoCellNode.m

```diff
@@ -125,56 +116,65 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   
-  ASLayoutSpec *spacer      = [[ASLayoutSpec alloc] init];    // FIXME: long locations overflow post time - set max size?
-  spacer.flexGrow           = YES;
-  spacer.flexShrink         = YES;
+  ASLayoutSpec *spacer           = [[ASLayoutSpec alloc] init];    // FIXME: long locations overflow post time - set max size?
+  spacer.flexGrow                = YES;
+  
+  UIEdgeInsets avatarInsets      = UIEdgeInsetsMake(HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER, HORIZONTAL_BUFFER);
+  ASInsetLayoutSpec *avatarInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:avatarInsets child:_userAvatarImageView];
 
   ASStackLayoutSpec *headerStack = [ASStackLayoutSpec horizontalStackLayoutSpec];
   headerStack.alignItems         = ASStackLayoutAlignItemsCenter;                     // center items vertically in horizontal stack
   headerStack.justifyContent     = ASStackLayoutJustifyContentStart;                  // justify content to the left side of the header stack
-  
-  UIEdgeInsets avatarInsets          = UIEdgeInsetsMake(HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER, HORIZONTAL_BUFFER);
-  ASInsetLayoutSpec *avatarInset     = [ASInsetLayoutSpec insetLayoutSpecWithInsets:avatarInsets child:_userAvatarImageView];
-  
-  headerStack.flexShrink = YES;
-
   [headerStack setChildren:@[avatarInset, headerSubStack, spacer, _photoTimeIntervalSincePostLabel]];
   
   // header inset stack
-  
   UIEdgeInsets insets                = UIEdgeInsetsMake(0, HORIZONTAL_BUFFER, 0, HORIZONTAL_BUFFER);
   ASInsetLayoutSpec *headerWithInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:insets child:headerStack];
-  headerWithInset.flexShrink = YES;
   
   // footer stack
-  
   ASStackLayoutSpec *footerStack     = [ASStackLayoutSpec verticalStackLayoutSpec];
   footerStack.spacing                = VERTICAL_BUFFER;
-  
   [footerStack setChildren:@[_photoLikesLabel, _photoDescriptionLabel, _photoCommentsView]];
   
   // footer inset stack
-  
   UIEdgeInsets footerInsets          = UIEdgeInsetsMake(VERTICAL_BUFFER, HORIZONTAL_BUFFER, VERTICAL_BUFFER, HORIZONTAL_BUFFER);
   ASInsetLayoutSpec *footerWithInset = [ASInsetLayoutSpec insetLayoutSpecWithInsets:footerInsets child:footerStack];
   
   // vertical stack
-  
   CGFloat cellWidth                  = constrainedSize.max.width;
   _photoImageView.preferredFrameSize = CGSizeMake(cellWidth, cellWidth);              // constrain photo frame size
   
   ASStackLayoutSpec *verticalStack   = [ASStackLayoutSpec verticalStackLayoutSpec];
   verticalStack.alignItems           = ASStackLayoutAlignItemsStretch;                // stretch headerStack to fill horizontal space
   [verticalStack setChildren:@[headerWithInset, _photoImageView, footerWithInset]];
-  verticalStack.flexShrink = YES;
 
   return verticalStack;
 }
 
 #pragma mark - Instance Methods
 
+- (void)fetchData
+{
+  [super fetchData];
+  
+  [_photoModel.commentFeed refreshFeedWithCompletionBlock:^(NSArray *newComments) {
```

Comment:
This is a very cool example of using fetch data!

It is probably worth adding a comment here, either linking to the documentation about intelligent preloading and overriding methods like this, or at least a call out to why this is simpler and also much more efficient then what can be easily implemented with UIKit.